prover/proof_submitter/valid_proof_submitter.go

```diff
@@ -54,6 +57,19 @@ func NewValidProofSubmitter(
 		return nil, err
 	}
 
+	var signalServiceNameBytes [32]byte
+	copy(signalServiceNameBytes[:], []byte("signal_service"))
+
+	l1SignalService, err := rpc.TaikoL1.Resolve0(nil, signalServiceNameBytes, false)
+	if err != nil {
+		return nil, err
+	}
+
+	l2SignalService, err := rpc.TaikoL1.Resolve0(nil, signalServiceNameBytes, false)
```

Comment:
should be `.Resolve(l2ChainId, signalServiceNameBytes` I think? this just gets the L1 signal service again

Reply:
changed to `rpc.TaikoL2.Resolve0(nil, signalServiceNameBytes, false)` locally, forgot to push it up...